Resolving data files: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3371/3371 [00:00<00:00, 26487.69it/s]
Found cached dataset imagefolder (/home/simtoon/.cache/huggingface/datasets/imagefolder/smtn_girls_likeOrNot-8f15de4de8339189/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f)
Loading cached split indices for dataset at /home/simtoon/.cache/huggingface/datasets/imagefolder/smtn_girls_likeOrNot-8f15de4de8339189/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-0b4a8a602117c2d7.arrow and /home/simtoon/.cache/huggingface/datasets/imagefolder/smtn_girls_likeOrNot-8f15de4de8339189/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-89c7b4019dbe549b.arrow
Train: 2696 | Test: 675







Downloading pytorch_model.bin: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 346M/346M [00:14<00:00, 23.2MB/s]
Some weights of ViTForImageClassification were not initialized from the model checkpoint at google/vit-base-patch16-224-in21k and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Traceback (most recent call last):
  File "/home/simtoon/git/attraction_classifier/train.py", line 38, in <module>
    trainingArgs = TrainingArguments(
                   ^^^^^^^^^^^^^^^^^^
  File "<string>", line 112, in __init__
  File "/home/simtoon/git/attraction_classifier/venv/lib/python3.11/site-packages/transformers/training_args.py", line 1372, in __post_init__
    and (self.device.type != "cuda")
         ^^^^^^^^^^^
  File "/home/simtoon/git/attraction_classifier/venv/lib/python3.11/site-packages/transformers/training_args.py", line 1795, in device
    return self._setup_devices
           ^^^^^^^^^^^^^^^^^^^
  File "/home/simtoon/git/attraction_classifier/venv/lib/python3.11/site-packages/transformers/utils/generic.py", line 54, in __get__
    cached = self.fget(obj)
             ^^^^^^^^^^^^^^
  File "/home/simtoon/git/attraction_classifier/venv/lib/python3.11/site-packages/transformers/training_args.py", line 1716, in _setup_devices
    raise ImportError(
ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.20.1`: Please run `pip install transformers[torch]` or `pip install accelerate -U`